modules/virtual_network_gateway/variables.tf

```diff
@@ -8,46 +15,16 @@ variable "location" {
   type        = string
 }
 
-variable "name_prefix" {
-  description = "A prefix added to all resource names created by this module"
-  default     = ""
-  type        = string
-}
-
-variable "name_suffix" {
-  description = "A suffix added to all resource names created by this module"
-  default     = ""
-  type        = string
-}
-
-variable "name" {
-  description = "The name of the Virtual Network Gateway. Changing this forces a new resource to be created"
-  type        = string
-}
-
 variable "tags" {
   description = "Azure tags to apply to the created resources."
   default     = {}
   type        = map(string)
 }
 
-variable "enable_zones" {
-  description = "If false, all the subnet-associated frontends and also all created Public IP addresses default to not to use Availability Zones (the `No-Zone` setting). It is intended for the regions that do not yet support Availability Zones."
-  default     = true
-}
-
-variable "avzones" {
-  description = <<-EOF
-  After provider version 3.x you need to specify in which availability zone(s) you want to place IP.
-  ie: for zone-redundant with 3 availability zone in current region value will be:
-  ```["1","2","3"]```
-  EOF
-  default     = []
-  type        = list(string)
-}
-
+# Virtual Network Gateway
 variable "type" {
   description = "The type of the Virtual Network Gateway. Valid options are Vpn or ExpressRoute. Changing the type forces a new resource to be created"
```

Comment:
maybe lets use `EOF` notation for wrapping descriptions?

Reply:
Same applies to other multi-sentence descriptions

Reply:
fixed